app/controllers/api/v2/hosts_controller.rb

```diff
@@ -262,6 +262,31 @@ def power
         end
       end
 
+      api :GET, '/hosts/:id/power_status', N_('Fetch the status of whether the vm is powered on or not')
```

Comment:
1. should be GET /hosts/ID/power ?
2. s/vm/host (as it shows bare metal ipmi power status too)

Reply:
Ah thanks, good catch.